components/prism-core.js

```diff
@@ -524,15 +524,21 @@ if (script) {
 	_.filename = script.src;
 
 	if (!_.manual && !script.hasAttribute('data-manual')) {
```

Comment:
This is only related to the intent of this PR but shouldn't `data-manual` set `Prism.manual`?

I think that it's really strange that you can include Prism via

```html
<script src="prism.js" data-manual></script>
```

And it will switch to manual highlighting without, well, being set to manual.

Reply:
It probably makes sense to set this flag when the HTML attribute is set to allow reading it from other code.